generators/app/templates/ui5.yaml

```diff
@@ -1,19 +1,31 @@
-specVersion: "2.0"
+specVersion: "3.0"
 metadata:
   name: <%= appId %>
 type: application
-#ui5Theme: sap_fiori_3
+framework:
+  name: <%= framework %>
+  version: "<%= frameworkVersion %>"
+  libraries: 
+    - name: sap.f
+    - name: sap.m
+    - name: sap.ui.core
+    - name: sap.ui.layout
+    - name: sap.ui.table
+    - name: themelib_<%= defaultTheme %>
+builder:
+  customTasks:
+    - name: ui5-tooling-transpile-task
+      afterTask: replaceVersion
 server:
   customMiddleware:
+  - name: ui5-tooling-transpile-middleware
+    afterMiddleware: compression
+  - name: ui5-middleware-livereload
```

Comment:
this should be always at then end to be the first middleware in the execution order

Reply:
Moved it to the end, was not aware of this